app/component/map/tile-layer/VectorTileLayerContainer.js

```diff
@@ -20,6 +21,15 @@ export default function VectorTileLayerContainer(props, { config }) {
         layersToAdd.push(CityBikes);
       }
 
+      if (
+        config.dynamicParkingLots &&
+        config.dynamicParkingLots.showDynamicParkingLots
+      ) {
+        layers.push(DynamicParkingLots);
+      }
+      if (config.bikeParks && config.bikeParks.show) {
```

Comment:
You need to change this code to take the new variable name `layersToAdd` into account. Also, please remove the code about dynamic parking lots from this PR.